src/libAtomVM/opcodesswitch.h

```diff
@@ -4119,6 +4420,8 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
                     SCHEDULE_NEXT(mod, INSTRUCTION_POINTER());
                     continue;
                 }
+
+                int orig_i = i;
```

Comment:
The purpose of this variable looks a bit obscure, maybe it deserve a comment here.

Reply:
done